### DISCUSSION TITLE
How complete is Unreal Engine 5 support?

### DISCUSSION THREAD

**Discussion by davidsanda** [nxrighthere/UnrealCLR#195] [General]
Tests seem to compile, but when changing test systems from default to some other enum, it enters into error state and all it does is throw error from screenshot.
So, how complete is the ue5 support?
![image](https://user-images.githubusercontent.com/37351867/139602196-c45ec066-2da9-45f2-8aa4-3d58b8cf4c1c.png)

**Comment by nxrighthere** [comment]
What you see on the screen is excepted output of the assertion test.

Unreal Engine 5 is backwards compatible with Unreal Engine 4 with minor changes, so the plugin support it out of the box.

**Comment by davidsanda** [comment]
Hi, you are quite right. Looking at it again, it works. One more thing that is confusing is this screenshot: 
![image](https://user-images.githubusercontent.com/37351867/139740291-24356b3d-b4fd-4837-b0ca-42432c45eb12.png)
from the https://www.youtube.com/watch?v=t8nP0IEWLZU

I cannot find anything like that in the level blueprint in the current git. Is it from older version, where this was exposed?

**Reply by nxrighthere** [reply]
This video is outdated, yes. Please, check out the [manual](https://github.com/nxrighthere/UnrealCLR/blob/master/MANUAL.md) to get familiar with the plugin.

**Reply by davidsanda** [reply]
Thanks for confirmation. I am new to unreal, so i was not sure, whether i overlooked something.
